DEVELOPING.md

```diff
@@ -16,11 +16,27 @@ The resulting image is called openjdk
 ### Cloud build
 To build using the [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/overview), you need to have the [Google Cloud SDK](https://cloud.google.com/sdk/) installed locally.
 ```
-$ ./scripts/cloudbuild.sh
+$ PROJECT_ID=my-project
+$ ./scripts/cloudbuild.sh gcr.io/$PROJECT_ID/openjdk
 ```
 
 If you would like to simulate the cloud build locally, pass in the `--local` argument.
 ```
 $ PROJECT_ID=my-project
-$ ./scripts/cloudbuild.sh --local
-```
\ No newline at end of file
+$ ./scripts/cloudbuild.sh gcr.io/$PROJECT_ID/openjdk --local
+```
+
+# Running Tests
+Integration tests can be run via [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/overview).
+These tests deploy a sample test application to App Engine using the provided runtime image, and 
+exercise various integrations with other GCP services.
+```bash
+$ PROJECT_ID=my-project
+$ ./scripts/integration_test.sh gcr.io/$PROJECT_ID/openjdk
+```
+
+To drive the integration tests using local cloud build, pass in the `--local` argument.
```

Comment:
It seems like there's little utility to the local run because the image under test still has to be on GCR.
I wonder if there is any way around it to support the case of running integration tests on the local image.